l10n_ch_dta/wizard/create_dta.py

```diff
@@ -724,10 +724,11 @@ def _create_dta(self, data):
                 record_type = RecordGt836
             elif country_code and country_code != 'CH':
                 record_type = RecordGt836
-            elif elec_pay == 'bvr':
+            #in v9 bv and bvr do not exist any more
+            elif elec_pay in ['bvr', 'postal']:
```

Comment:
AFAIK `elec_pay` can't be `'bvr'`. Simply check that it is equal to `'postal'` and remove `'bvr'`